backend/core/src/listings/combined/combined-listing.entity.ts

```diff
@@ -330,18 +389,76 @@ export class CombinedListing {
   @ViewColumn({ name: "last_application_update_at" })
   lastApplicationUpdateAt: Date
 
+  /*
   @ViewColumn()
   county: string
 
   @ViewColumn()
   city: string
+  */
 
   @ViewColumn()
   neighborhood: string
 
   @ViewColumn({ name: "reserved_community_type_name" })
   reservedCommunityTypeName: string
 
+  // The lowest monthly rent across all units
+  @ViewColumn({ name: "min_monthly_rent" })
+  minMonthlyRent: number
+
+  // The highest monthly rent across all units
```

Comment:
In this case, if a building has a studio/SRO for $500/mo, and a 4 bedroom for $4500/mo 

if I search for `4 bedrooms`+ `<$501/mo` will I get this listing as a result?

Reply:
Yep I remember that's what I thought. 

I'll add this to the PM/UX meeting for next week to see if they can come up with some UI treatment around this. I know that it was originally discussed to be this way with PM/UX but I haven't seen it portrayed as a fuzzy match in the mocks so far so just want to make sure that they're on the same page.